ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>OUTPUT_FILE_URI = "gs://automl-text-notebook-20210604194135/output/prediction-e2e-text-classification-model-2021-06-04T20:13:37.743910Z/predictions_00001.jsonl"</blockquote><p>Suggest renaming the value to something like "[output-file-uri]"</p><p> </p><p> </p><p>I also got the following error when running this snippet:</p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
&lt;ipython-input-36-cd9d86fcc90b&gt; in &lt;module&gt;
      5 get_ipython().system(' pip install --user jsonlines')
      6 
----&gt; 7 import jsonlines
      8 
      9 with jsonlines.open(f"data/{FILE_NAME}") as reader:

ModuleNotFoundError: No module named 'jsonlines'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
Done (I hope). I added the call to `pip install jsonlines`.

Reply:
Thanks, I assume the issue is something with my setup.